src/DotLiquid/NamingConventions/RubyNamingConvention.cs

```diff
@@ -25,6 +25,11 @@ public StringComparer StringComparer
             get { return StringComparer.OrdinalIgnoreCase; }
         }
 
+        public StringComparer ConditionComparer
+        {
+            get { return StringComparer.OrdinalIgnoreCase; }

```

Comment:
I'll switch the Ruby ConditionalComparer to Ordinal then - conditions will have to use the correct capitalization.